src/server/search/search_family_test.cc

```diff
@@ -287,4 +287,23 @@ TEST_F(SearchFamilyTest, SimpleUpdates) {
   }
 }
 
+TEST_F(SearchFamilyTest, Unicode) {
+  EXPECT_EQ(Run({"ft.create", "i1", "schema", "title", "text", "visits", "numeric"}), "OK");
+
+  // Explicitly using screaming uppercase to check utf-8 to lowercase functionality
+  Run({"hset", "d:1", "title", "Веселая СТРЕКОЗА Иван", "visits", "400"});
+  Run({"hset", "d:2", "title", "Die fröhliche Libelle Günther", "visits", "300"});
+  Run({"hset", "d:3", "title", "השפירית המהירה יעקב", "visits", "200"});
+  Run({"hset", "d:4", "title", "πανίσχυρη ΛΙΒΕΛΛΟΎΛΗ Δίας", "visits", "100"});
```

Comment:
Greek! Means strong dragonfly zeus 🤣 did you google translate this? (loved ALL the easter egg 👀 )

Reply:
Yes, I was just mixing adjective + dragonfly + some stereotypical name 🙂 Sounds like children stories